packages/apps/graph-client/src/graphql/queries.graph.ts

```diff
@@ -97,6 +97,251 @@ export const getRecentHeights: DocumentNode = gql`
   }
 `;
 
+export const getAccount: DocumentNode = gql`
+  query getAccount($moduleName: String!, $accountName: String!) {
+    account(moduleName: $moduleName, accountName: $accountName) {
+      # id
+      accountName
+      moduleName
+
+      totalBalance
+
+      chainAccounts {
+        # accountName
+        balance
+        chainId
+        # guard {
+        #   keys
+        #   predicate
+        # }
+        # moduleName
+        # transactions
+        # transfers
+      }
+      transactions {
+        edges {
+          node {
+            # id
+            # badResult
+            chainId
+            code
+            # continuation
+            creationTime
+            # data
+            # gas
+            # gasLimit
+            # gasPrice
+            # goodResult
+            height
+            # logs
+            # metadata
+            # nonce
+            # eventCount
+            # pactId
+            # proof
+            requestKey
+            # rollback
+            # senderAccount
+            # step
+            # ttl
+            # transactionId
+          }
+        }
+      }
+      transfers {
+        edges {
+          node {
+            amount
+            # blockHash
+            chainId
+            senderAccount
+            height
+            # id
+            # idx
+            # moduleHash
+            # moduleName
+            requestKey
+            receiverAccount
+          }
+        }
+      }
+    }
+  }
+`;
+
+export const getChainAccount: DocumentNode = gql`
+  query getChainAccount(
+    $moduleName: String!
+    $accountName: String!
+    $chainId: String!
+  ) {
+    chainAccount(
+      moduleName: $moduleName
+      accountName: $accountName
+      chainId: $chainId
+    ) {
+      accountName
+      balance
+      chainId
+      guard {
+        keys
+        predicate
+      }
+      moduleName
+      transactions {
+        edges {
+          node {
+            # id
+            # badResult
+            chainId
+            code
+            # continuation
+            creationTime
+            # data
+            # gas
+            # gasLimit
+            # gasPrice
+            # goodResult
+            height
+            # logs
+            # metadata
+            # nonce
+            # eventCount
+            # pactId
+            # proof
+            requestKey
+            # rollback
+            # senderAccount
+            # step
+            # ttl
+            # transactionId
```

Comment:
After merging this and the latest open PR's maybe it would be a good idea to create aliases for the fields as we did with `CoreEventFields`